libs/user/user.ts

```diff
@@ -320,36 +379,52 @@ class User {
     );
   }
 
+  /**
+   * Allow user to check the deposits that haven't been processed yet
+   *
+   * @async
+   * @method checkPendingDeposits
+   * @returns {Promise} - This promise resolves into an object which contains the pendind deposits and their balances
+   */
   async checkPendingDeposits() {
     return this.client.getPendingDeposits(this.zkpKeys);
   }
 
+  /**
+   * Allow user to get the total Nightfall Layer2 balance of its commitements
+   *
+   * @async
+   * @method checkNightfallBalances
+   * @returns {Promise} - This promise resolves into an object which contains the addresses and balances for the Layer2 addresses sent as a parametar.
+   */
   async checkNightfallBalances() {
     return this.client.getNightfallBalances(this.zkpKeys);
   }
 
   /**
+   * Allow user to check the balance of the pending spent commitments on Layer2
    *
-   * @method checkLayer2PendingSpentBalances should get return the balance of pending spent commitments
-   * from transfer and withdraw for each ERC address
-   * @param {string[]} ercList - an array of ERC smart contracts
-   * @param {boolean} shouldFilterByCompressedZkpPublicKey - a boolean value that will define in the endpoint if the query
-   * @returns
+   * @async
+   * @method checkPendingTransfers
+   * @returns {Promise}  - This promise resolves into an object whose properties are the
+    addresses of the ERC contracts of the tokens held by this account in Layer 2. The
+    value of each propery is the number of tokens pending spent (transfer & withdraw)
+    from that contract. 
```

Comment:
And about this tbh I can't comment. I thought this was all about "transfers" but apparently it is also related to "withdrawals" (?) So still need to get my head around it

Reply:
This is written by @luizoamorim . Not really sure.

Reply:
I got this comments from nf3 project.